tutorials/finetune_lora.md

```diff
@@ -59,25 +59,21 @@ python finetune/lora.py --quantize "bnb.nf4-dq"
 
 The table below lists a comparison with different settings on a StableLM 3B model finetuned with LoRA on Alpaca for 5,000 iterations using a microbatch size of 4:
 
-| Settings                                                | Training Memory  | Training Time  | Loss      | Inference Memory |
```

Comment:
I haven't been able to reproduce these memory values even after checking the original PR that added them, but this PR's values are consistent with current main

Reply:
Did you also check whether peak memory usage is consistent with main? Because that would be important regarding instantiation + quantization.

Reply:
That is what I meant with

> this PR's values are consistent with current main